model/work.py

```diff
@@ -141,13 +141,21 @@ class Work(Base):
     presentation_edition_id = Column(Integer, ForeignKey('editions.id'), index=True)
 
     # One Work may have many associated WorkCoverageRecords.
-    coverage_records = relationship("WorkCoverageRecord", backref="work")
+    coverage_records = relationship(
+        "WorkCoverageRecord", backref="work",
+        cascade="all, delete-orphan"
+    )
 
     # One Work may be associated with many CustomListEntries.
+    # However, a CustomListEntry may lose its Work without
+    # ceasing to exist.
```

Comment:
This one tripped me up so I added a clarifying comment. An example would be an item on the NYT best-sellers list. It's associated with a Work in our system, but then we lose our license to the Work and it's deleted. That book is still on the best-sellers list!

Reply:
Isn't it a problem that it's still on that best-sellers list?

Reply:
Not really; the NYT dataset is just a bunch of information about some books. There's no requirement that we have licenses for any of those books. If we have a Work that corresponds to an item in the NYT dataset, and then we lose the license, the NYT dataset shouldn't change.

Similarly if we start out _not_ having a license for a book in the NYT dataset, and we get a license, the new Work should become associated with the NYT dataset.